donut/modules/directory_search/templates/search_results.html

```diff
@@ -1,91 +1,107 @@
 {% extends 'layout.html' %}
 {% block page %}
-<div class='container theme-showcase jumbotron' style='float: none; padding-bottom: 25px'>
-	<a class='btn btn-info' href='{{ url_for("directory_search.directory_search") }}'>Back</a>
-	<div>Your search returned {{ total }} results</div>
-	{% if users %}
-		<label for='per_page'>Results per page</label>
-		<select id='per_page'>
-			{% for i in range(10, 50, 10) %}
-				<option value={{ i }} {% if i == per_page  %} selected {% endif %}>{{ i }}</option>
-			{% endfor %}
-		</select>
-
-		<table class='table table-striped table-condensed'>
-			<thead>
-				<tr>
-					<th>Name</th>
-					<th>Graduation Year</th>
-					{% if show_images %}
-						<th> Image </th>
-					{% endif %}
-				</tr>
-			</thead>
-			<tbody>
-				{% for user in users %}
-					<tr>
-						<td>
-							<a href='{{ url_for("directory_search.view_user", user_id=user["user_id"]) }}'>{{ user['full_name'] }}</a>
-						</td>
-						<td>{{ user['graduation_year'] or '' }}</td>
-						{% if show_images %}
-							<td>
-								{% if user['image'] %}
-									<img src='{{ url_for("directory_search.get_image", user_id=user["user_id"]) }}' style='max-width: 100px; max-height: 150px'/>
-								{% endif %}
-							</td>
-						{% endif %}
-					</tr>
+	<div class='container theme-showcase jumbotron' style='float: none; padding-bottom: 25px'>
+		<a class='btn btn-info' href='{{ url_for("directory_search.directory_search") }}'>Back</a>
+		<div>Your search returned {{ total }} results</div>
+		{% if users %}
+			<label for='per_page'>Results per page</label>
+			<select id='per_page'>
+				{% for i in range(10, 50, 10) %}
+					<option value={{ i }} {% if i == per_page  %} selected {% endif %}>{{ i }}</option>
 				{% endfor %}
-			</tbody>
-		</table>
-		<ul class="pagination">
-			{% set num_pgs = total // per_page + (1 if total % per_page > 0 else 0) %}
-			<li class="page-item {% if page == 1 %} disabled {% endif %}">
-				<a class="page-link" {% if page != 1 %} href="{{ url_for('directory_search.search', page=page-1, per_page=per_page, **query_info) }}" {% endif %} aria-label="Previous">
-					<span aria-hidden="true">&laquo;</span>
-					<span class="sr-only">Previous</span>
-				</a>
-			</li>
-			{% if page - 6 > 1 %}
-				<li class="page-item">
-					<a class="page-link" href="{{ url_for('directory_search.search', page=1, per_page=per_page, **query_info) }}">1</a>
-				</li>
-				<li class="page-item">
-					<a class="page-link">...</a>
-				</li>
+			</select>
+
+			{% if show_images %}
+				<div class="box">
+					{% for user in users %}
+						<div>
+						<img src={% if user['image'] %} '{{ url_for("directory_search.get_image", user_id=user["user_id"]) }}' {% else %} '{{ url_for("static", filename="images/donut_man.png") }}' {% endif %} style='width: 100px; height: 150px; object-fit: cover'/>
```

Comment:
This line is very long; can you split the attributes across a few lines? Also potentially use a `class` for styling instead of inline `style`?
Love the donut stick figure tho lol